Content.Shared/Weapons/Melee/Components/MeleeThrowOnHitComponent.cs

```diff
@@ -42,6 +42,20 @@ public sealed partial class MeleeThrowOnHitComponent : Component
     /// </summary>
     [DataField, AutoNetworkedField]
     public bool ActivateOnThrown;
+
+    /// <summary>
+    /// Whether the item has already landed, preventing it from hitting more people off-cooldown.
+    /// </summary>
+    [DataField, AutoNetworkedField]
```

Comment:
```suggestion
    [DataField]
```
oh, and remove the autonetworking since it's not used anyways